doc/title.xml

```diff
@@ -22,7 +22,7 @@
   <Author>Ben Steinberg</Author>
   <Author>Finn Smith</Author>
   <Author>Jhevon Smith</Author>
-  <Author>Michael Torpey</Author>
+  <Author>Michael Young</Author>
```

Comment:
It would be good if you could maintain the alphabetical order of this list, and more precisely, to put Michael at the end of the list now 🙂

Reply:
Ah yes, of course!